share/completions/adb.fish

```diff
@@ -49,7 +49,7 @@ function __fish_adb_run_command -d 'Runs adb with any -s parameters already give
 end
 
 function __fish_adb_list_packages
-    __fish_adb_run_command pm list packages 2>/dev/null | string replace 'package:' ''
+    __fish_adb_run_command pm list packages 2\>/dev/null | string replace 'package:' ''
```

Comment:
Please add a comment explaining this cursedness.

Something along the lines of "we need to redirect *inside* the adb shell, because it sends the stderr to its stdout".

Reply:
I wrote `# Escape > to redirect stderr inside adb shell, because it sends stderr of the command to its stdout.` but forgot about actually pushing it and now the commit has been merged. Oh well.

Reply:
Oh, no worries. I added explanation later in 9ae880f6a.